test/recipes/80-test_cmp_http.t

```diff
@@ -31,8 +31,8 @@ plan skip_all => "These tests are not supported in a no-ec build"
 plan skip_all => "These tests are not supported in a no-sock build"
     if disabled("sock");
 
-plan skip_all => "Tests involving local HTTP server not available on Windows, AIX or VMS"
-    if $^O =~ /^(VMS|MSWin32|AIX)$/;
+plan skip_all => "Tests involving local HTTP server not available on Windows, AIX, VMS, or NonStop ia64"
+    if $^O =~ /^(MSWin32|AIX|VMS|nonstop-nse)$/;
```

Comment:
Ah, I'll have to remember to check if VMS can deal with this test...  I see no *immediate* reason why not...

Reply:
There was a different issue that motivated exclusion.
As far as I remember, there was an issue with the shell/system commands used by the Perl script to start and stop the test server.